src/components/layouts/Table/TableBody.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const TableBody: React.FC<{ className?: string }> = ({ className = '', children }): JSX.Element => {
+  return <tbody className={`bg-white divide-y divide-gray-200 ${className}`}>{children}</tbody>;
+};
```

Comment:
A bit shorter solution:
`const TableBody: React.FC<{ className?: string }> = ({ className = '', children }): JSX.Element => (
  <tbody className={`bg-white divide-y divide-gray-200 ${className}`}>{children}</tbody>;
);`

Also what I like to do is configure an interface for each component with the same naming. But for one parameter I don't have problems using this style.

Reply:
Yeah as it's only className thought it was unnecessary
however we should keep the tbody & the class otherwise no point